packages/fx-core/scripts/download-templates-zip.js

```diff
@@ -20,14 +20,6 @@ const scenrioSupportedLanguages = new Map([
     ["tab", "default", "frontend"],
     ["js", "ts", "csharp"],
   ],
-  [
-    ["bot", "default", "bot"],
-    ["js", "ts", "csharp"],
-  ],
-  [
-    ["msgext", "default", "bot"],
-    ["js", "ts"],
-  ],
```

Comment:
If all files are moved from bot folder to bot-msg, maybe we can still use bot folder and treat this code as default bot code?

Reply:
Yes. I have updated this.